README.md

```diff
@@ -140,6 +140,22 @@ func application(_ application: UIApplication, didFailToRegisterForRemoteNotific
 }
 ```
 
+### Location pushes
+
+Starting with iOS 15, Apple supports power efficient way to request location trough [location push service extension](https://developer.apple.com/documentation/corelocation/creating_a_location_push_service_extension).
+
+To use this new functionality you should obtain Apple entitlments, create location push service extension and make some tweaks in your push notifications code within your app:
+
+* Add Apple location pushes entitlments into your project ("AblyLocationPush" folder).
+
+* In addition to saving your regular device token by calling `ARTPush.didRegisterForRemoteNotifications(withDeviceToken:realtime:)` you should request another device token specifically for the location pushes purpose. You do so by calling `CLLocationManager.startMonitoringLocationPushes(completion:)` within a `ARTPushRegistererDelegate.didActivateAblyPush(:)` delegate callback or whenever is appropriate in your app.
```

Comment:
> within a `ARTPushRegistererDelegate.didActivateAblyPush(:)` delegate callback or whenever is appropriate in your app.

This is a confusing statement. Which one is it — it needs to be done inside `ARTPushRegistererDelegate.didActivateAblyPush(:)`, or the user can do it whenever they want?

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/794e634e187b3df8f0a56ffac1243e41e592211e